src/main/java/uk/gov/hmcts/ccd/domain/enablingcondition/jexl/JexlEnablingConditionParser.java

```diff
@@ -54,7 +54,11 @@ public Boolean evaluate(String enablingCondition, Map<String, ?> caseEventData)
             if (expression != null) {
                 JexlScript expressionScript = engine.createScript(expression);
                 Map<String, Object> data = retrieveContextData(caseEventData, expressionScript.getVariables());
-                return (Boolean) expressionScript.execute(new MapContext(data));
+                if (data.isEmpty()) {
+                    return true;
+                } else {
```

Comment:
this may have serious regression effect on other apps. We need to think again further and in more details.